### PR TITLE
rename connectors guide

### PR DESCRIPTION
**Pull request by christinaausley**
## What is the purpose of the change

Rename Connectors guide to `Implementing out-of-the-box Connectors` to avoid confusion.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
@akeller I know https://github.com/camunda/camunda-platform-docs/pull/1205 might cause a merge conflict here but I'll resolve 👍

File: docs/guides/introduction-to-camunda-platform-8.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ After getting started with Camunda Platform 8, dive deeper in our next steps sec
 - [Set up your first development project](./setting-up-development-project.md) - Set up your first project to model, deploy, and start a process instance.
 - [Set up client connection credentials](./setup-client-connection-credentials.md) - Create, name, and connect your client.
 - [Automate a process using BPMN](./automating-a-process-using-bpmn.md) - Learn more about the mechanics and elements of BPMN, and build your first BPMN diagram.
-- [Implement connectors](./implementing-connectors.md) - Automate complex business processes by inserting them into BPMN diagrams within Web Modeler.
+- [Implement connectors](./configuring-out-of-the-box-connectors.md) - Automate complex business processes by inserting them into BPMN diagrams within Web Modeler.
```
Comment (MaxTru) [inline docs/guides/introduction-to-camunda-platform-8.md:43]: Why not
```suggestion
- [Configuring an out-of-the-box connector](./configuring-out-of-the-box-connectors.md) - Automate complex business processes by inserting them into BPMN diagrams within Web Modeler.
```

File: versioned_docs/version-8.0/guides/introduction-to-camunda-platform-8.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ After getting started with Camunda Platform 8, dive deeper in our next steps sec
 - [Set up your first development project](./setting-up-development-project.md) - Set up your first project to model, deploy, and start a process instance.
 - [Set up client connection credentials](./setup-client-connection-credentials.md) - Create, name, and connect your client.
 - [Automate a process using BPMN](./automating-a-process-using-bpmn.md) - Learn more about the mechanics and elements of BPMN, and build your first BPMN diagram.
-- [Implement connectors](./implementing-connectors.md) - Automate complex business processes by inserting them into BPMN diagrams within Web Modeler.
+- [Implement connectors](./configuring-out-of-the-box-connectors.md) - Automate complex business processes by inserting them into BPMN diagrams within Web Modeler.
```
Comment (MaxTru) [inline versioned_docs/version-8.0/guides/introduction-to-camunda-platform-8.md:43]: Why not
```suggestion
- [Configuring an out-of-the-box connector](./configuring-out-of-the-box-connectors.md) - Automate complex business processes by inserting them into BPMN diagrams within Web Modeler.
```

**Review by MaxTru — CHANGES_REQUESTED** [review]

File: docs/guides/introduction-to-camunda-platform-8.md (hunk 1 above)
Comment (christinaausley) [reply docs/guides/introduction-to-camunda-platform-8.md:43]: Nice catch. Adjusting in commit.

File: versioned_docs/version-8.0/guides/introduction-to-camunda-platform-8.md (hunk 2 above)
Comment (christinaausley) [reply versioned_docs/version-8.0/guides/introduction-to-camunda-platform-8.md:43]: Nice catch. Adjusting in commit.

**Review by akeller — DISMISSED** [review]

**Comment by akeller** [comment]
@christinaausley I've approved both of your Connectors PRs. Please merge them in the order you prefer 😄

**Review by MaxTru — APPROVED** [review]

**Pull request merged**